### ISSUE TITLE
AutoRun on Save ?

### ISSUE THREAD

**Issue by pcarbonn** [haskelly-dev/Haskelly#76]
It would be great to run my Haskell program immediately on Save, in the REPL.  Can that be done ?

**Comment by pcarbonn** [comment]
Many thanks !

Please note that the exact configuration parameter should be `--fast --file-watch --exec myproject`, where myproject is the name of my executable.

Unfortunately, this creates a full build, and so it is not very fast.  I'm going to use [ghcid](https://github.com/ndmitchell/ghcid) instead.

**Comment by martrik** [comment]
Yes you are right, my suggested params would only build, not run on-save. ghcid seems a good solution!
